docker-compose.yml

```diff
@@ -49,6 +54,8 @@ services:
       - "slurmdbd"
     networks:
       - slurm-network
+    environment:
+      - REQUESTS_VERIFY_SSL=false
```

Comment:
let's not set it here 
if needed for some test, better to set within a single test